numpy/testing/utils.py

```diff
@@ -1178,9 +1178,11 @@ def assert_raises(*args,**kwargs):
     return nose.tools.assert_raises(*args,**kwargs)
 
 
-def assert_raises_regex(exception_class, expected_regexp,
-                        callable_obj=None, *args, **kwargs):
+def assert_raises_regex(exception_class, expected_regexp, *args, **kwargs):
     """
+    assert_raises_regex(exception_class, expected_regexp, callable_obj=None,
```

Comment:
I cleaned up the docstring, and added the two function signatures from assertRaisesRegex:
https://docs.python.org/3/library/unittest.html#unittest.TestCase.assertRaisesRegex